service_account_getter.go

```diff
@@ -0,0 +1,114 @@
+package kubeauth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+	kubeerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const annotationKeyPrefix = "auth-metadata.vault.hashicorp.com/"
```

Comment:
There are a couple of other k8s integrations that use Vault-like URLs - please could we verify that this is consistent to some degree with Agent injector's annotations and VSO? They are serving different purposes, but I'm not certain how consistently we use vault.hashicorp.com for the domain.

Reply:
Thank you. I've updated the prefix to `vault.hashicorp.com/alias-metadata-`.